Apps/Sandcastle/index.html

```diff
@@ -21,6 +21,7 @@
     <script src="jsHintOptions.js"></script>
     <script src="gallery/gallery-index.js"></script>
     <script src="CesiumSandcastle.js"></script>
+    <script src="../../Source/Cesium.js"></script>
```

Comment:
Oh, it's not the prebuilt copy.  There's a reference already to `Source/Cesium.js` at the top of `CesiumSandcastle.js`.  You may need to change the way the variable is bound since the definition might be deferred.